### PR TITLE
Add Default Prefix to New Fee Estimation Constant Names

### PR DESCRIPTION
**Pull request by tholonious**

### CODE REVIEW

**Comment by tholonious** [comment]
> [!WARNING]
> <b>This pull request is not mergeable via GitHub because a downstack PR is open. Once all requirements are satisfied, merge this PR as a stack <a href="https://app.graphite.dev/github/pr/deso-protocol/core/1256?utm_source=stack-comment-downstack-mergeability-warning" >on Graphite</a>.</b>
> <a href="https://graphite.dev/docs/merge-pull-requests">Learn more</a>

* **#1256** <a href="https://app.graphite.dev/github/pr/deso-protocol/core/1256?utm_source=stack-comment-icon" target="_blank">Graphite</a> 👈
* **#1252** <a href="https://app.graphite.dev/github/pr/deso-protocol/core/1252?utm_source=stack-comment-icon" target="_blank">Graphite</a>: 1 other dependent PR ([#1255](https://github.com/deso-protocol/core/pull/1255) <a href="https://app.graphite.dev/github/pr/deso-protocol/core/1255?utm_source=stack-comment-icon" target="_blank">Graphite</a>)
* **#1253** <a href="https://app.graphite.dev/github/pr/deso-protocol/core/1253?utm_source=stack-comment-icon" target="_blank">Graphite</a>
* `feature/proof-of-stake`

This stack of pull requests is managed by Graphite. <a href="https://stacking.dev/?utm_source=stack-comment">Learn more about stacking.</a>
<h2></h2>

 Join @tholonious and the rest of your teammates on <a href="https://graphite.dev?utm-source=stack-comment">Graphite <b>Graphite</b></a>

File: lib/constants.go [hunk 1]
```diff
@@ -1353,7 +1359,7 @@ var DeSoMainnetParams = DeSoParams{
 
 	// The congestion factor determines when we will start to increase or decrease fees.
 	// We set the congestion factor to 90% for past blocks and mempool. This makes it so that we will
-	// start to increase fees when the past N blocks (DefaultMempoolFeeEstimatorNumPastBlocks) are
+	// start to increase fees when the past N blocks (DefaultMempoolPastBlocksCongestionFactorBasisPoints) are
```
Comment (lazynina) [inline lib/constants.go:1362]:
```suggestion
	// start to increase fees when the past N blocks (DefaultMempoolFeeEstimatorNumPastBlocks) are
```

**Review by lazynina — APPROVED** [review]
Just a nit in the comments, but otherwise LGTM.

File: lib/constants.go (hunk 1 above)
Comment (tholonious) [reply lib/constants.go:1362]: Resolved in https://github.com/deso-protocol/core/pull/1261. Thanks for fixing this

**Comment by tholonious** [comment]
### Merge activity

* **Apr 22, 10:39 PM EDT**: @tholonious started a stack merge that includes this pull request via [Graphite](https://app.graphite.dev/github/pr/deso-protocol/core/1256).

**Pull request merged**